CHANGELOG.md

```diff
@@ -52,6 +52,19 @@ Swift 3.0
   Similarly, the new `RecoverableError` and `CustomNSError` protocols
   allow additional control over the handling of the error.
 
+* [SE-0095](https://github.com/apple/swift-evolution/blob/master/proposals/0095-any-as-existential.md):
+The `protocol<...>` composition construct has been removed. In its place, an infix type operator `&` has been introduced.
+
+```swift
+let a: Foo & Bar
+let b = value as? A & B & C
+func foo<T : Foo & Bar>(x: T) { … }
+func bar(x: Foo & Bar) { … }
+typealias G = GenericStruct<Foo & Bar>
+```
+
+The empty protocol composition, the `Any` type, was previously defined as being `protocol<>`. This has been removed from the standard library and `Any` is now a keyword with the same behaviour.
+
```

Comment:
Please format this in a way that is consistent with the existing README updates.